app/components/arclight/document_component.html.erb

```diff
@@ -11,7 +11,7 @@
     <%= document.normalized_title %>
   <% end %>
   <%= render 'arclight/requests', document: document %>
-  <%= render Arclight::BookmarkComponent.new document: document, action: bookmark_config %>
+  <%= render (blacklight_config.index.document_actions.arclight_bookmark_control.component || Arclight::BookmarkComponent).new document: document, action: bookmark_config %>
```

Comment:
```suggestion
  <%= render (presenter&.view_config&.document_actions&.arclight_bookmark_control.component || Arclight::BookmarkComponent).new document: document, action: bookmark_config %>
```

I think the presenter should have the appropriate configuration context, maybe?

Reply:
Hey @cbeer, I don't seem to be finding what I'm looking for using the presenter view_config

<img width="754" alt="image" src="https://github.com/user-attachments/assets/abcfcd3a-1d8c-49e4-ae29-355b1eee95fe">